cmd/govim/config/config.go

```diff
@@ -6,6 +6,20 @@ const (
 	InternalFunctionPrefix = "_internal_"
 )
 
+type EnvVar string
+
+const (
+	// EnvVarGovimUseGoplsFromPath is an environment variable which, when set to
+	// the value "true", configures govim to use the gopls binary found in PATH
+	// instead of the version required by the govim module.
+	//
+	// **************** WARNING ****************
+	//
+	// Use of this environment variable comes with no warranty, because we have
+	// no guarantees that the gopls version found in PATH works with govim.
+	EnvVarGovimUseGoplsFromPath = "GOVIM_USE_GOPLS_FROM_PATH"
```

Comment:
Fixed the const name; but I'd like to leave the env var. I find it frustrating to have environment variables that don't namespace what/where they are used. I like the Go environment variables for that reason. I added underscores here because of the length of the name... but maybe they could go for consistency with the Go approach

Reply:
Yes, I was only talking about the variable name.